corehq/apps/app_manager/tests/test_bulk_app_translation.py

```diff
@@ -215,6 +231,20 @@ def test_missing_itext(self):
         except Exception as e:
             self.fail(e)
 
+    def test_bad_column_name(self):
+        self.upload_raw_excel_translations(self.upload_headers_bad_column,
+            self.upload_data,
+            expected_messages=[
+                u'Sheet "module1_form1" has less columns than expected. Sheet '
+                'will be processed but the following translations will be '
+                'unchanged: default_fra',
+                u'Sheet "module1_form1" has unrecognized columns. Sheet will '
+                'be processed but ignoring the following columns: default-fra',
+                u'App Translations Updated!'
```

Comment:
I had hoped that HQ would accept either form: "default-fra" or "default_fra". The meaning of either column name seems pretty unambiguous, and it seems like a perfect place to apply the [robustness principle](https://en.wikipedia.org/wiki/Robustness_principle). Is there a reason you decided against that?